src/cmd/create.go

```diff
@@ -426,6 +426,7 @@ func createContainer(container, image, release, authFile string, showCommandToEn
 		"--volume", homeDirMountArg,
 		"--volume", toolboxPathMountArg,
 		"--volume", runtimeDirectoryMountArg,
+		"--env", fmt.Sprintf("HOME=%s", currentUser.HomeDir),
```

Comment:
Nitpick: could you please move this further up between `--dns` and `--env`?

I suppose this will bake `HOME` into the `podman inspect` JSON as `/home/foo`?  Just like we do for `TOOLBOX_PATH` and `XDG_RUNTIME_DIR`?

If so, then we need to remember that there are two processes with two different UIDs that are started by Toolbx inside the containers.  The first is the entry point of the container that runs as `root:root` and the other is the process running as the user's UID that the user actually interacts with.  Currently, the entry point has `HOME` set to `/home/foo` on the first start, and then `/root` for subsequent starts.

I am worried that baking `HOME` to `/home/foo` will make the entry point (running as `root:root`) always use it as it's home directory.  If the entry point ends up creating some files in its home directory as a side-effect, these will pollute the user's home directory too, and, potentially, even conflict with the user's own files.

This isn't a huge problem because this is already the case for the first start, so, I suppose things are working alright at the moment.  However, it will entrench it a bit more.

Since we are already trying to clean up this weird oddity with the `HOME` environment variable, I think we should also take the step to ensure that the entry point running as `root:root` has `HOME` set to the correct value for `root`.